dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/UserDAO.java

```diff
@@ -76,6 +76,11 @@ public User findByGitHubUsername(String username) {
         return (User)query.uniqueResult();
     }
 
+    public Long findPublishedEntries(String username)  {
+        final Query query = namedQuery("io.dockstore.webservice.core.User.countPublishedEntries").setParameter("username", username);
```

Comment:
the way this query is written (a count), I don't think you will ever get a null result so I think you can cast to/return a long